src/behaviors/eventbrite/eventbrite.js

```diff
@@ -29,6 +29,14 @@ class EventBrite extends Behavior {
   }
 
   execute(command, message, channel, messageData) {
+
+   this.bot.postMessage(channel, `Sorry guys, I'm good at dropping beatz, not numbers.. I'll look around for you though okay? Here's a picture! https://goo.gl/DWWqs4`, {
```

Comment:
Extra period after numbers.

Reply:
Added another period to demonstrate ellipsis...